src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowMonitor.scala

```diff
@@ -56,7 +71,32 @@ class WorkflowMonitor(parent: ActorRef, pollInterval: Duration, workflow: Workfl
     }
   }
 
-  def onWorkflowDone(updateWorkflow: Workflow, txn: RawlsTransaction): Unit = {
-    // call any code to update entity model here
+  def withMethodConfig(txn: RawlsTransaction)(op: MethodConfiguration => (Workflow, Option[Map[String, Attribute]])): (Workflow, Option[Map[String, Attribute]]) = {
```

Comment:
Is there a reason why we can't replace all the instances of ```(Workflow, Option[Map[String, Attribute]])``` with ```WorkflowStatusChange``` ? It's clearer and less verbose.